worlds/shapez/docs/en_shapez.md

```diff
@@ -0,0 +1,63 @@
+# shapez
+
+## What is this game?
+
+shapez is an automation game about cutting, rotating, stacking, and painting shapes, that you extract from randomly
+generated patches on an infinite canvas, and sending them to the hub to complete levels. The "tutorial", where you
+unlock a new building or game mechanic (almost) each level, lasts until level 26, where you unlock freeplay with 
+infinitely more levels, that require a new, randomly generated shape. Alongside the levels, you can unlock upgrades,
+that make your buildings work faster.
+
+## Where is the options page?
+
+The [player options page for this game](../player-options) contains all the options you need to configure
+and export a config file.
+
+## What does randomization do to this game?
+
+Buildings and gameplay mechanics, that you normally unlock by completing a level, and upgrade improvements are put 
+into the item pool of the multiworld. Also, if enabled, the requirements for completing a level or buying an upgrade are
+randomized.
+
+## What is the goal of shapez in Archipelago?
+
+As the game has no actual goal where the game ends, there are (currently) 4 different goals you can choose from in the 
+player options:
+1. Vanilla: Complete level 26 (the end of the tutorial).
+2. MAM: Complete an in the player options specified level after level 26. It's recommended to build a 
```

Comment:
```suggestion
2. MAM: Complete a player-specified level after level 26. It's recommended to build a 
```

Less is more, this reads a little cleaner. I don't speak German, so I leave it to your judgment if a similar change should be applied to the DE documents (or even here for that matter).